assets/scripts/main.js

```diff
@@ -30,6 +30,10 @@ import './dialogs/_save_as_image'
 
 // Street
 import './streets/name'
+import './streets/scroll'
+
+// Gallery
+import './gallery/scroll'
```

Comment:
There is also no `import './util/fetch_blocking.js'` but there are no imports from `./util` at all and `util/fetch_blocking.js` doesn't contain any side effects so maybe it was inteded.